### DISCUSSION TITLE
Wideo not watched in whole is hidden as it would be watched in full, bug or feature?

### DISCUSSION THREAD

**Discussion by DDzwiedziu** [FreeTubeApp/FreeTube#6354] [Q&A]
I have turned on the option to hide videos from the feed that are already watched. However if I interrupt watching at any point by either changing the view or crashing the system, the video, that I was in middle of watching, will become treated as “watched” and hidden.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @DDzwiedziu this isnt a bug. The moment you decide to watch a video it will be marked as watched. There are issues open to improve this but they are a low priority https://github.com/FreeTubeApp/FreeTube/issues/609 https://github.com/FreeTubeApp/FreeTube/issues/964
